### PR TITLE
Compile ReactOS with -fno-common

### PR DESCRIPTION
**Pull request by Extravert-ir**
That is a default option on modern compilers

### CODE REVIEW

File: sdk/cmake/gcc.cmake [hunk 1]
```diff
@@ -39,7 +39,8 @@ if(STACK_PROTECTOR)
 endif()
 
 # Compiler Core
-add_compile_options(-pipe -fms-extensions -fno-strict-aliasing)
+# note: -fno-common is default since GCC 10
```
Comment (HBelusca) [inline sdk/cmake/gcc.cmake:42]: Is it supported by clang as well?

File: modules/rostests/apitests/compiler/ms/seh/seh0001.c [hunk 2]
```diff
@@ -6,7 +6,6 @@
 #include "seh.h"
 
 char test[] = "SEH0001.c";
-int perfect;
```
Comment (HBelusca) [inline modules/rostests/apitests/compiler/ms/seh/seh0001.c:9]: I wonder what was their idea behind this.
Do you think it's possible to send that upstream?

**Review by HBelusca — COMMENTED** [review]
Looks nice; does the PR fix all our codebase or is there other modifications to be done?

**Comment by Extravert-ir** [comment]
> Some wine code modifs that may be worth sending upstream.

Wine is already compiled like that

File: sdk/cmake/gcc.cmake (hunk 1 above)
Comment (Extravert-ir) [reply sdk/cmake/gcc.cmake:42]: Yes

File: modules/rostests/apitests/compiler/ms/seh/seh0001.c (hunk 2 above)
Comment (Extravert-ir) [reply modules/rostests/apitests/compiler/ms/seh/seh0001.c:9]: No, I have already spent enough time on this :P

**Pull request merged**